src/common/vector/value_vector.cpp

```diff
@@ -182,6 +182,13 @@ void ArrowColumnVector::setArrowColumn(ValueVector* vector, std::shared_ptr<arro
     arrowColumnBuffer->column = std::move(column);
 }
 
+void ArrowColumnVector::setArrowColumn(
```

Comment:
Remove the above function `ArrowColumnVector::setArrowColumn(ValueVector* vector, std::shared_ptr<arrow::Array> column)`.